spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /posts' do
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done but `index` and `show` actions for `User` and `Post` controllers should be tested like that 🙏🏼 

```ruby
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
```